projects/collector-v1.md

```diff
@@ -0,0 +1,42 @@
+### Description
+
+Quite a few of our users are happy using a non-v1 Collector, as evidenced by talks delivered by eBay or VTEX. On the other hand, other companies have internal policies forbidding the usage of software that hasn’t reached v1.
+
+Other companies, hearing the success stories of the “early adopters”, tried to use the Collector, but due to a lack of guidance, ended up using components that weren’t ready for prime time and were frustrated.
+
+Our main goal with this project is to define what's the scope for Collector v1, defining which use-cases we want to satisfy first, followed by stabilizing the components that are required for that. Once we are there, we'll focus on documentation for those components and use cases.
+
+We believe we are close to a v1 of the Collector, given that users are using it in production already. Therefore, we are confident we can deliver a v1 this year.
+
+Teams involved in this:
+* @open-telemetry/collector-approvers 
+* @open-telemetry/collector-contrib-approvers 
+* @open-telemetry/collector-maintainers 
+* @open-telemetry/collector-contrib-maintainer 
+ 
+### Project Board
+
+To be created.
+
+### Deliverables
+
+The current goal (subject to change) is to have deliverables covering our two current distributions: core and contrib. Deliverables should be linked from the main opentelemetry.io website. 
+
+#### Must-haves
+* Binaries for Windows, MacOS, and Linux for amd64 and x86
+* Multi-arch container images
+
+#### Nice to have
+* OS-specific packages for Linux (deb, rpm, …)
```

Comment:
It would be nice to add homebrew if we have time/someone does the work, but I think the general sentiment across Collector approvers and maintainers is that we already have a lot on our plate to go through and stabilize and we should keep the scope small and focused if we want to deliver this in a reasonable timeframe

Reply:
@mx-psi that's why I added it to the nice to have, but we can skip it entirely :)

Reply:
uh, didn't see that somehow :smile: Fine for me to add it there then, yes :)